addons/resource.language.en_gb/resources/strings.po

```diff
@@ -6811,12 +6811,7 @@ msgctxt "#13516"
 msgid "Add art"
 msgstr ""
 
-#empty strings from id 13517 to 13549
-
-#: system/settings/settings.xml
-msgctxt "#13550"
```

Comment:
this is still needed: https://github.com/MartijnKaijser/xbmc/blob/setting_cleanup/system/settings/settings.xml#L1882

this is a videoscreen setting now